jmx-metrics/src/main/resources/target-systems/kafka.groovy

```diff
@@ -14,81 +14,178 @@
  * limitations under the License.
  */
 
-def messagesInPerSec = otel.mbean("kafka.server:type=BrokerTopicMetrics,name=MessagesInPerSec")
-otel.instrument(messagesInPerSec, "kafka.messages.in", "number of messages in per second",
-        "1", "Count", otel.&longValueCallback)
+def messagesInPerSec = otel.mbean("kafka.server:type=BrokerTopicMetrics,name=MessagesInPerSec,topic=*")
+otel.instrument(messagesInPerSec,
+  "kafka.message.count",
+  "The number of messages received by the broker",
+  "messages",
+  [
+    "topic" : { mbean -> mbean.name().getKeyProperty("topic") },
+  ],
+  "Count", otel.&longCounterCallback)
+
+def requests = otel.mbeans(["kafka.server:type=BrokerTopicMetrics,name=TotalProduceRequestsPerSec,topic=*",
+                           "kafka.server:type=BrokerTopicMetrics,name=TotalFetchRequestsPerSec,topic=*"])
+otel.instrument(requests,
+  "kafka.request.count",
+  "The number of requests received by the broker",
+  "requests",
+  [
+    "topic" : { mbean -> mbean.name().getKeyProperty("topic") },
+    "type" : { mbean -> switch(mbean.name().getKeyProperty("name")) {
+      case "TotalProduceRequestsPerSec":
+        return "Produce"
```

Comment:
nit, do these need to be capitalized?

Reply:
I'll lowercase all attribute values. This was intended to match values coming directly from mbean properties in other metrics but I'll lowercase them as well.